.github/CODEOWNERS

```diff
@@ -1,6 +1,6 @@
 *                   @confluentinc/cli
-internal/pkg/flink/ @confluentinc/cloud-surfaces
-internal/pkg/ccloudv2/flink.go @confluentinc/cli @confluentinc/cloud-surfaces
-internal/pkg/ccloudv2/flink_gateway.go @confluentinc/cli @confluentinc/cloud-surfaces
+pkg/flink/ @confluentinc/cloud-surfaces
+pkg/ccloudv2/flink.go @confluentinc/cli @confluentinc/cloud-surfaces
+pkg/ccloudv2/flink_gateway.go @confluentinc/cli @confluentinc/cloud-surfaces
```

Comment:
the complete path would be `"github.com/confluentinc/cli/v3/pkg/flink"`. Is this the right syntax?